src/Files.App/Dialogs/AddItemDialog.xaml.cs

```diff
@@ -27,7 +27,7 @@ public AddItemDialog()
 			InitializeComponent();
 		}
 
-		public new async Task<DialogResult> ShowAsync() => (DialogResult)await base.ShowAsync();
+		public new Task<DialogResult> ShowAsync() => base.ShowAsync().AsTask().ContinueWith(t => (DialogResult)t.Result);
```

Comment:
I don't like this kind of change (i.e. `ContinueWith` instead of chained `await`) as it reduces the readability a lot. There're also many such changes in other places through this PR.
/cc: @yaira2